libs/components/forms/src/lib/modules/checkbox/checkbox-group.component.html

```diff
@@ -25,6 +27,14 @@
       [popoverContent]="helpPopoverContent"
     />
   </legend>
+  <div skyId #hintTextEl="skyId">
```

Comment:
You can get rid of the nested `div`s here by using `ngClass` to assign the CSS classes to the `div` when hint text exists (note I added `sky-margin-stacked-lg` so you can delete the SCSS file below).

```
<div
  skyId
  [ngClass]="{
    'sky-font-deemphasized sky-margin-stacked-lg sky-checkbox-group-hint-text': !!hintText
  }"
  #hintTextEl="skyId"
>
  {{ hintText }}
</div>
```

Reply:
updated